tdrs-backend/tdpservice/users/models.py

```diff
@@ -139,3 +167,8 @@ def region(self, value):
             raise ValidationError(
                 _("Data Analyst cannot have a region assigned to them"))
         self.location = value
+
+    @property
```

Comment:
TIL about `@property`, I've just always been using getters and function calls. Neat!